doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -1057,21 +360,6 @@ cJSON
 Documentation
 =============
 
-* Added:
-
-  * Documentation page about :ref:`ug_matter_device_security` in the Matter protocol section.
-  * Documentation template for the :ref:`Ecosystem integration <Ecosystem_integration>` user guides.
-  * Documentation on :ref:`ug_avsystem`.
-  * The :ref:`ug_nrf70_developing` user guide.
-  * A page on :ref:`ug_nrf70_features`.
-  * Documentation template for :ref:`Applications <application>`.
-  * The :ref:`ug_nrf5340_gs` guide.
-
-* Updated:
-
-  * The :ref:`software_maturity` page with details about Wi-Fi feature support.
-  * The :ref:`app_power_opt` user guide by adding sections about power saving features and PSM usage.
-  * The :ref:`ug_thingy53` by aligning with current simplified configuration.
-  * The :ref:`ug_nrf52_developing` by aligning with current simplified FOTA configuration.
+|no_changes_yet_note|
 
 .. |no_changes_yet_note| replace:: No changes since the latest |NCS| release.
```

Comment:
May be this could be moved to shortcuts.txt.

Reply:
moved